src/styles/block/card.scss

```
@@ -0,0 +1,127 @@
+@import "./src/styles/mixins";
+@import "./src/styles/variables";
+
+.main {
+  @include display-justify-content;
+}
+
+.catalog {
+  width: 1024px;
+  @include display-justify-content;
+  flex-wrap: wrap;
+  padding: 0 40px;
+  margin: 0 auto;
+}
+
+.card {
+  width: 200px;
+  height: 403px;
+  padding: 30px 14px 15px;
+  margin-bottom: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  &__photo {
+    margin: 0 auto 42px;
+  }
+  &__photo-img {
+    width: 160px;
+    height: 134px;
+  }
+  &__content {
+    margin-left: 14px;
+    margin-right: 14px;
+  }
+  &__title {
+    margin-bottom: 5px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+  }
+  &__code {
+    margin-bottom: 13px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 11px;
+    line-height: 13px;
+  }
+  &__rate {
+    display: flex;
+    height: 19px;
+    margin-bottom: 18px;
+    align-items: flex-end;
+  }
+  &__stars {
+    display: flex;
+    margin-right: auto;
+    margin-left: 1px;
+  }
+  &__star {
+    flex-basis: 21px;
+    padding-right: 6px;
+  }
+  &__rewiew {
+    position: relative;
+    bottom: 3px;
+    align-items: baseline;
+    flex-basis: auto;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 11px;
+    line-height: 13px;
+  }
+  &__price {
+    display: flex;
+    height: 15px;
+    margin-bottom: 24px;
+    align-items: baseline;
+  }
+  &__price-info {
+    font-size: 11px;
+    flex-basis: 50%;
+  }
+  &__price-amount {
+    flex-basis: 50%;
+    display: flex;
+    margin-bottom: -1px;
+    flex-direction: row-reverse;
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
```

```suggestion
  &__price {
    display: flex;
    height: 15px;
    margin-bottom: 24px;
    justify-content: space-between;
    align-items: baseline;
  }
  &__price-info {
    font-size: 11px;
  }
  &__price-amount {
    font-size: 16px;
    font-weight: 700;
```надмірне використання flex, старайся застосовувати flex до батьківського контейнера і вирівняти flex-items.